lib/src/interfaces/Options.ts

```diff
@@ -167,12 +181,17 @@ export interface OptionsTopBarLargeTitle {
    */
   color?: Color;
   /**
-   * Set the font family of large title's text
+   * Set the font family for a text
```

Comment:
Out of curiosity, why was this description changed? Seems like the previous phrasing was more explicit.

Reply:
Copy & paste hack job 😢. I copied the `fontFamily`, `fontStyle` and `fontWeight` to every interface that needed it and didn't circle back to change the comments. Do you want that changed?